### ISSUE TITLE
libavif.pas fails to compile (mode Delphi)

### ISSUE THREAD

**Issue by klaus101** [bgrabitmap/bgrabitmap#239]
bgrabitmap-11.5.8, Win 10, Laz. 3.0.
For a program once converted from Delphi, trying to do some bgrabitmap operations here now,  I had compiler errors on libwebp.pas and saw this link:   https://github.com/bgrabitmap/bgrabitmap/issues/235
Tried  the  fix herein (it’s already part of the dev version) and it worked.

After that I had another compiler complaint:
libavif.pas(157,41) Error: Syntax error, ")" expected but ":=" found
// Line 157 is a type definition:        avifPlanesFlag = (AVIF_PLANES_YUV := 1 shl 0,AVIF_PLANES_A := 1 shl 1,

After some puzzling I thought it might be due to MDelphi too, and tried:

```
    unit libavif;
     
    {$mode objfpc}
```

All compiled fine now.
You might consider if it’s worthy to pick this up.

**Comment by klaus101** [comment]
Hello circular,

thank you for your attention!
I tested with the recent dev-package as just picked up, and the compilation did run flawlessly now.
From this it can be closed :-)

Tested with: Win 10 64 bit, Lazarus 3.2 FPC 3.2.2, bgrabitmap used via sub-folder within the app.

I was happy that, after libwebp.pas, only one other file (libavif.pas) was concerned, so I assumed to apply the same fix should be harmless.
